markdown/report.Rmd

```diff
@@ -62,7 +142,7 @@ glossary2 <- c("Palliative care is an approach that improves the quality of life
 glossary <- data.frame(glossary1, glossary2)
```

Comment:
I noticed for the last item, "SG website" has a hyperlink attached. Not sure if it's possible to attach it in the table, can look into it!